install.bash

```diff
@@ -14,6 +14,7 @@ common=(
 )
 
 if [ `uname` == 'Darwin' ]; then
+  /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
ah hah! 👍

Reply:
I switched to a new laptop today, kinda noticed the old good brew was missing... ;-)